### ISSUE TITLE
Questions Arising...

### ISSUE THREAD

**Issue by ghost** [yariplus/minecraft-nodebb-integration#36]
Does this do group sync at all?

**Comment by yariplus** [comment]
I apologise many of the features are broken. I am doing my best to work
through the current issues.

The plugin currently has real time updates to the status widget, as well as
chat synchronization.

Thank you for trying out my plugin, I will try to get your issues resolved
soon.

**Comment by ghost** [comment]
Yes, I do really love it. My apologies for being so mean, it's just I was so excited someone actually paid attention to NodeBB and appreciated integrated Minecraft... 

So thank you for all of your efforts. I do remember using this a while ago and it worked beautifully. I have also had some issues just keep tracking of all the Node updates, let alone NodeBB etc...

Thank you!
